packages/components/src/box-control/index.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+/**
```

Comment:
In most files in the project, I see a space between the last import in a group and the next dependency group comment. It would be good to follow this trend for consistency.